test/compilable/test20530b.d

```diff
@@ -0,0 +1,27 @@
+module test.compilable.test20530b;
```

Comment:
I would prefer extending the original test case instead of scattering the tests across multiple files. This makes it easier to find missing test cases (and might reduce the runtime of the test suite)

Reply:
Ok, still gonna need a new file to test another bug that i'm attaching.